openQA.spec

```diff
@@ -177,6 +177,8 @@ Requires:       perl(LWP::UserAgent)
 Requires:       perl(Mojolicious)
 Requires:       perl(Regexp::Common)
 Requires:       perl(Try::Tiny)
+Requires:       python3-future
```

Comment:
Please move this into a special subpackage. the openQA client doesn't need python3

Reply:
So... will there be a separate package for Python utilities then? The last update didn't address this but there wasn't a response to this comment either.

I'm not sure if there was a reason to use Python here over Perl. I love Python, but I second that the dependency isn't very appealing when the client doesn't use any Python.

Reply:
fixed, please see the updated changes. Moved into the subpackage "python-scripts" now.